build.sbt

```diff
@@ -2,12 +2,26 @@ import java.nio.file.Files
 
 name := "codacy-scalastyle"
 
-scalaVersion := "2.13.5"
+ThisBuild / scalaVersion := "2.13.5"
 
-libraryDependencies ++= Seq(
+val scalastyleVersion = "1.5.0"
+
+val commonDeps = Seq(
   "org.scala-lang.modules" %% "scala-xml" % "1.3.0",
-  "com.beautiful-scala" %% "scalastyle" % "1.5.0",
+  "com.beautiful-scala" %% "scalastyle" % scalastyleVersion,
   "com.codacy" %% "codacy-engine-scala-seed" % "5.0.3"
 )
 
+libraryDependencies ++= commonDeps
+
+val `doc-generator` = project
+  .settings(
+    libraryDependencies ++= commonDeps ++ Seq("com.lihaoyi" %% "os-lib" % "0.7.3"),
```

Comment:
Seems strange to have one way of IO on the main code, and other on the docs generator. Is there an advantage of bringing this dependency? Is it something that you want to start to introduce for us to use it? You owe us as Lunch & Learn if you add it :D

Reply:
Deal :D

Reply:
Changed the code to use `better-files` that we were already depending on instead of `os-lib`